arm-compute/2016-03-30/swagger/containerService.json

```diff
@@ -352,19 +380,23 @@
           },
           "required": [
             "name",
-            "dnsPrefix"
+            "dnsPrefix",
+            "count",
```

Comment:
"count" is not a required property for AgentPoolProfile.  This should be removed.

https://msdn.microsoft.com/en-us/library/azure/mt711471.aspx